wordpress-mobile/gutenberg-mobile
# Focus on title when creating a new post.

## Description:

Resolves https://github.com/wordpress-mobile/WordPress-iOS/issues/10594

This PR focuses the title when opening Gutenberg, if the post has no title and no content.  This is a simplified and initial approach, that seems to be enough to match the condition that we're dealing with a new post.

If we find this condition isn't enough we can expand incrementally.

## Related PRs:

Gutenberg: https://github.com/WordPress/gutenberg/pull/13874

## Testing:

### Test 1:

1. Run the demo app.

Make sure the focus is NOT in the title, since it's not empty and there are blocks.

### Test 2:

1. Edit `App.js`, and force `initialTitle = '';` and `initialData = '';`.
2. Run the demo app.
3. Make sure the title and content are empty.  If not, go back to step 1.

Make sur the focus is in the title, since the title is empty and there are no blocks.
